### PR TITLE
Set texture name

### PR DESCRIPTION
**Pull request by slightHYP**
### Reference Discussion #3132

Added the possibility to set a custom name to the model-viewer texture object.
The model-viewer image name has been set to the file name (plus extension) if the source field is valorized. 

The CreateTextures example has been updated in order to display these new values.

image

### CODE REVIEW

File: packages/model-viewer/src/features/scene-graph/image.ts [hunk 1]
```diff
@@ -71,8 +73,13 @@ export class Image extends ThreeDOMElement implements ImageInterface {
     return this.uri != null ? 'external' : 'embedded';
   }
 
+  set name(name: string) {
+    (this[$sourceObject] as GLTFImage).name = "image_" + name;
```
Comment (elalish) [inline packages/model-viewer/src/features/scene-graph/image.ts:77]: Do we want to prepend `image_` here? Seems odd to set one name and get something different back.

File: packages/modelviewer.dev/examples/scenegraph/index.html [hunk 2]
```diff
@@ -16,614 +16,578 @@
...
   <link type="text/css" href="../../styles/docs.css" rel="stylesheet" />
-  <link rel="shortcut icon" type="image/png" href="../../assets/favicon.png"/>
-
+  <link rel="shortcut icon" type="image/png" href="../../assets/favicon.png" />
   <!-- 💁 OPTIONAL: The :focus-visible polyfill removes the focus ring for some input types -->
   <script src="../../node_modules/focus-visible/dist/focus-visible.js" defer></script>
-
   <script>
-    window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
+    window.ga = window.ga || function () { (ga.q = ga.q || []).push(arguments) }; ga.l = +new Date;
```
Comment (elalish) [inline packages/modelviewer.dev/examples/scenegraph/index.html:31]: I'm guessing you had auto-format set for this file. Can you revert it so our formatting stays? I know it's not great, but it's the only way we've found to keep the example text from having a bunch of tabs in front of every line. Also, I can't really see your changes when the diff is this bad.

**Review by elalish — CHANGES_REQUESTED** [review]
Thanks! Just a couple small things...

File: packages/model-viewer/src/features/scene-graph/image.ts (hunk 1 above)
Comment (slightHYP) [reply packages/model-viewer/src/features/scene-graph/image.ts]: Yep, this is a typo from development tests. Removed on next commit

File: packages/modelviewer.dev/examples/scenegraph/index.html (hunk 2 above)
Comment (slightHYP) [reply packages/modelviewer.dev/examples/scenegraph/index.html]: I noticed this also, reverted on next commit

**Review by elalish — APPROVED** [review]
Thanks!

**Pull request merged**